package Exception;

public class B{
	public static void main(String[] args) {
	try {
    int i = 10;
     System.out.println(i);  
	}
	   catch(Throwable moneky) {
		  B ref = null;
		   System.out.println("ref.i");
	   }
     
	
     }
}
